perf.groovy

```diff
@@ -0,0 +1,120 @@
+                    "py \"${perfWorkingDirectory}\\Microsoft.BenchView.JSONFormat\\tools\\build.py\" git --branch %GIT_BRANCH_WITHOUT_ORIGIN% --type ${runType} --output \"${perfWorkingDirectory}\"")
+                    batchFile("py \"%WORKSPACE%\\Microsoft.BenchView.JSONFormat\\tools\\machinedata.py\" --output \"${perfWorkingDirectory}\"")
+
+			        // Build solution and run the performance tests
+				    batchFile("\"%WORKSPACE%\\build.cmd\" -perf /p:PerfIterations=10 /p:PerfOutputDirectory=\"${perfWorkingDirectory}\" /p:PerfCollectionType=stopwatch")
+
+			       //Create submission json and upload to Benchview
+				   batchFile("for /f \"tokens=*\" %%a in ('dir /b/a-d ${perfWorkingDirectory}\\*.xml') do (py \"${perfWorkingDirectory}\\Microsoft.BenchView.JSONFormat\\tools\\measurement.py xunitscenario \"${perfWorkingDirectory}\\%%a\" --better desc --output \"${perfWorkingDirectory}\")")
+                   batchFile("py \"${perfWorkingDirectory}\\Microsoft.BenchView.JSONFormat\\tools\\submission.py\" \"${perfWorkingDirectory}\\measurement.json\"" +
+                   "--build \"${perfWorkingDirectory}\\build.json\"" +
+                   "--machine-data \"${perfWorkingDirectory}\\machinedata.json\"" +
+                   "--metadata \"${perfWorkingDirectory}\\submission-metadata.json\"" +
+                   "--group \"SDK Perf Tests\"" +
+                   "--type \"${runType}\"" +
+                   "--config-name \"${configuration}\"" +
+                   "--config Configuration \"${configuration}\"" +
+                   "--architecture \"${arch}\"" +
+                   "--machinepool \"perfsnake\"" +
+				   "--output \"${perfWorkingDirectory}\"")
+                   batchFile("py \"${perfWorkingDirectory}\\Microsoft.BenchView.JSONFormat\\tools\\upload.py \"${perfWorkingDirectory}\submission.json\" --container stability")
+                }
+            }
+
+            def archiveSettings = new ArchivalSettings()
+            archiveSettings.addFiles('${perfWorkingDirectory}/**')
+            archiveSettings.setAlwaysArchive()
+            Utilities.addArchival(newJob, archiveSettings)
+            Utilities.standardJobSetup(newJob, project, isPR, "*/${branch}")
+
+            newJob.with {
+                logRotator {
+                    artifactDaysToKeep(30)
+                    daysToKeep(30)
+                    artifactNumToKeep(200)
+                    numToKeep(200)
+                }
+                wrappers {
+                    timeout {
+                        absolute(240)
+                    }
+                }
+            }
+
+            if (isPR) {
+                TriggerBuilder builder = TriggerBuilder.triggerOnPullRequest()
+                builder.setGithubContext("${os} ${arch} SDK Perf Tests")
+
+                builder.triggerOnlyOnComment()
+			    //Phrase is "test Windows_NT x64 perf"
+                builder.setCustomTriggerPhrase("(?i).*test\\W+${os}\\W+${arch}\\W+perf.*")
```

Comment:
This should include something about SDK in the trigger phrase

Reply:
Done...